### ISSUE TITLE
Odonatologica: font issue, no space between token

### ISSUE THREAD

**Issue by myrmoteras** [gsautter/goldengate-imagine#386]
I am not sure whether this is the same issue as https://github.com/gsautter/goldengate-imagine/issues/370

![image](https://user-images.githubusercontent.com/4609956/33805139-00ea6534-ddb4-11e7-9161-1c39c7d449ab.png)

produces this text:
Theforest-coveredlowlandsoftheKikoriRiverBasininsouthernPapuaNew Guineaextend fromthecoastal regionand deltainland to theextensive limestoneplainsoftheDaraiPlateau.Theodonatesofthisregionweredocumented
         
[10.5281zenodo.1040322_Theischinger_et_al_2017.pdf](https://github.com/gsautter/goldengate-imagine/files/1545654/10.5281zenodo.1040322_Theischinger_et_al_2017.pdf)

**Comment by gsautter** [comment]
Nope, this is not the same as 370. This is simply too high a threshold for putting a space between words on conversion to plain text. I've already lowered this threshold (which works relative to the word height), so this should come out file right with the next build.
